WordPress/src/main/java/org/wordpress/android/ui/posts/GutenbergWarningFragmentDialog.kt

```diff
@@ -0,0 +1,124 @@
+package org.wordpress.android.ui.posts
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import android.support.v7.app.AppCompatDialogFragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.CheckBox
+import org.wordpress.android.R
+import org.wordpress.android.widgets.WPTextView
+
+/**
+ * Basic dialog fragment with support for 1,2 or 3 buttons.
```

Comment:
Since we're on a PR again about this, let's change this comment which I think was a leftover from back when the dialog was way more generic. WDYT?

Reply:
Oh! Nicely spotted - changed the comment to reflect the real purpose of the class in 368239ddc0